config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/config/loader/policy/PolicyConverterRegistry.java

```diff
@@ -12,18 +12,34 @@
 import javax.inject.Inject;
 import javax.inject.Singleton;
 import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 import static java.util.Collections.unmodifiableSet;
 
 @Singleton
 public class PolicyConverterRegistry {
     private final Set<PolicyConverter> policyConverters;
+    private static final Logger LOGGER = Logger.getLogger(PolicyConverterRegistry.class.getName());
 
     @Inject
     public PolicyConverterRegistry(final Set<PolicyConverter> converters) {
         this.policyConverters = unmodifiableSet(converters);
     }
 
+    @NotNull
+    public boolean isValidPolicyExtension(String fileName) {
+        boolean isValidPolicyExtension = policyConverters.stream()
+            .anyMatch(converter -> fileName.endsWith(converter.getPolicyTypeExtension()));
+
+        if (isValidPolicyExtension) {
```

Comment:
really minor, but you can skip the else if you do something like this:
```
if (!isValidPolicyExtension) {
    Log here
}

return isValidPolicyExtension;
```

Reply:
Agreed